modules/core/src/main/scala/org/tessellation/Main.scala

```diff
@@ -59,9 +63,20 @@ object Main
           cfg.snapshot
         )
 
+        _ <- GlobalSnapshotEventsPublisher
```

Comment:
don't you mind moving it into AppResources or some kind of nested dependency? I'd like to avoid what we have on the current Mainnet - the mess in the main initialisation flow because of hundreds of dependencies that are created in a single flow

Reply:
So that's an interesting case. There's only a single method that needs to get called upon the initialization so it's very similar to the `Daemon` that has only the `start` method. Maybe I should represent it then as a `Daemon`?